### ISSUE TITLE
Poor performance on the price network

### ISSUE THREAD

**Issue by senwev** [preddy5/A-Deep-Generative-Model-for-Graph-Layout#1]
Hello Dear contributor,
I have tried your codes with graph can_96 , and it shows a perfect result.But when I changed the dataset to another graph (such as 96 nodes price network graph generated by graph-tool),it receive poor performance.
Have you tried some other graphs except can_96?I would like to see your result on other networks.

**Comment by preddy5** [comment]
Can you described, what steps you followed to test your results on the new graph?

**Comment by senwev** [comment]
Up to now, I have tested 2 graph structures, price_96(generated by `g=graph-tool.price_network(96,derected=False)`) and lesmis (generated by `graph_io.load_graph("path/graphs/lesmis.vna")`).
Their layout datasets were generated by your `create_data.py` file and all configrations were set same with yours except some neccessary changes related to number of nodes.
After 100 epochs of training, price_network got a bad results while lesmis had a better results .I guess that's probably because price_network has more equivalent nodes while current codes does not include Gromov Wesserstein Distance process.
Thanks for your reply!

**Comment by preddy5** [comment]
@senwev I agree with your hypothesis that the difference could be because of GM distance. Since it works well on the other dataset.
